app/forms/update_user_password_form.rb

```diff
@@ -0,0 +1,26 @@
+class UpdateUserPasswordForm
+  include ActiveModel::Model
+  include FormPasswordValidator
+
+  attr_accessor :password
+  attr_reader :user
+
+  def persisted?
+    true
+  end
+
+  def initialize(user)
+    @user = user
+  end
+
+  def submit(params)
+    self.password = params[:password]
+
+    return false unless valid?
+    @user.update(params)
+  end
+
+  def mobile_changed?
+    false
+  end
```

Comment:
ok -- do you think this is "good enough" for now, or do you think it's important to refactor before landing?